plugins/modules/net_tools/nmcli.py

```diff
@@ -2099,15 +2099,18 @@ def _compare_conn_params(self, conn_info, options):
                     # MAC addresses are case insensitive, nmcli always reports them in uppercase
                     value = value.upper()
                     # ensure current_value is also converted to uppercase in case nmcli changes behaviour
-                    current_value = current_value.upper()
+                    if (current_value):
```

Comment:
Please remove the unnecessary braces here (and below).

Reply:
Done. Sorry for not reading the contribution guidelines with enough attention.

Reply:
No worries (besides this probably isn't mentioned there, this is mainly something Python specific - Python isn't C where such braces are necessary).